src/lucky_record/associations.cr

```diff
@@ -172,7 +172,9 @@ module LuckyRecord::Associations
       {% nilable = false %}
     {% end %}
 
-    column {{ assoc_name.id }}_id : Int32{% if nilable %}?{% end %}
```

Comment:
So this hardcoded `Int32` was blowing up. I haven't worked with any lucky_record associations yet really, so I'm not aware of all the use cases. But I was able to fix this one by fetching the `id` column type from the "owning" model.

Reply:
This makes perfect sense 👍 

I think it may be more resilient and a bit faster to have a `PRIMARY_KEY_TYPE` constant on the model so you can shorten it to `model.resolve.constant(:PRIMARY_KEY_TYPE)` or something. What do you think?

Reply:
Checks out, thanks for the tip 👍